environments/storage/file-uploader/amplify/backend/readme.md

```diff
@@ -0,0 +1 @@
+This is a readme file to retrigger CICD
```

Comment:
We can delete this file after 7 days, when the last github action cache expires. The last github action cache is missing the storage files. So that's why this file was created so a new hash could be generated.